packages/editor/src/utils/media-upload/index.js

```diff
@@ -1,7 +1,12 @@
+/**
+ * External dependencies
+ */
+import { noop, uniqueId } from 'lodash';
```

Comment:
I know there is an effort to move away from lodash in the packages ( https://github.com/WordPress/gutenberg/issues/17025 ). Not sure if that applies here by but wanted to mention it.

Reply:
removed `noop`, I think we still use uniqueId from lodash? Or should I use `import { v4 as uuid } from 'uuid';` here?
`

Reply:
We're no longer using Lodash in the repository, and #17025 has been closed. I'd recommend `uuid` or even React's [useId](https://react.dev/reference/react/useId) hook that we no support since we upgrade to React 18.